src/ontology/mondo-edit.obo

```diff
@@ -179366,7 +179366,9 @@ subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "Giedion syndrome" RELATED [GARD:0007800]
 synonym: "Sugio-Kajii syndrome" EXACT [DOID:14743]
+synonym: "trichorhinophalangeal dysplasia (syndrome) types 1/3" EXACT [PMID:28426188]
 synonym: "trichorhinophalangeal dysplasia type I" RELATED [DOID:14743, MTH:NOCODE]
+synonym: "trichorhinophalangeal dysplasia types 1/3" EXACT [PMID:28426188]
 synonym: "trichorhinophalangeal syndrome type 1" RELATED [GARD:0007800]
```

Comment:
the ~type 1 (different spelling) are both exact and related synonyms. I will update this.